extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -48,11 +54,68 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
 
     // Create a paywall dialog element that have required attributes to be able to be
     // rendered by amp-subscriptions.
-    // TODO(#37285): complete the rest of paywall dialog UI based on the publisher-provided attributes.
-    const dialogEl = (
-      <div subscriptions-dialog subscriptions-display="NOT granted"></div>
+    this.dialogEl_ = (
+      <div subscriptions-dialog subscriptions-display="NOT granted">
+        <div class="i-amphtml-story-subscriptions-banner"></div>
+        <div class="i-amphtml-story-subscriptions-title"></div>
+        <div class="i-amphtml-story-subscriptions-subtitle-first"></div>
+        <div class="i-amphtml-story-subscriptions-subtitle-second"></div>
+        <div class="i-ampthml-story-subscriptions-button-container">
+          <div
+            class="i-ampthml-story-subscriptions-button-google"
+            subscriptions-action="subscribe"
+            subscriptions-display="NOT granted"
+            subscriptions-service="subscribe.google.com"
+            subscriptions-decorate
+          >
+            Subscribe
+          </div>
+          <div
+            class="i-amphtml-story-subscriptions-button-publisher"
+            subscriptions-action="subscribe"
+            subscriptions-display="NOT granted"
+          >
+            <div class="publisher-button-text"></div>
+            <img class="publisher-logo"></img>
+          </div>
+        </div>
+        <div class="i-amphtml-story-subscriptions-signin">
+          Already a subscriber?
+          <a
+            subscriptions-action="login"
+            subscriptions-display="NOT granted"
+            style="text-decoration: underline; font-weight: bold;"
+          >
+            Sign in
+          </a>
+        </div>
+      </div>
+    );
+
+    this.applyAttributeAsText_('title', '.i-amphtml-story-subscriptions-title');
+    this.applyAttributeAsText_(
+      'publisher-button-text',
+      '.i-amphtml-story-subscriptions-button-publisher .publisher-button-text'
+    );
+    this.applyAttributeAsText_(
+      'banner-text',
+      '.i-amphtml-story-subscriptions-banner'
+    );
+    this.applyAttributeAsText_(
+      'subtitle-first',
+      '.i-amphtml-story-subscriptions-subtitle-first'
+    );
+    this.applyAttributeAsText_(
+      'subtitle-second',
+      '.i-amphtml-story-subscriptions-subtitle-second'
+    );
+    const logoImg = dev().assertElement(this.dialogEl_.querySelector('img'));
+    logoImg.setAttribute(
+      'src',
+      this.element.getAttribute('publisher-logo-url')
```

Comment:
Please set all of these directly from the JSX template. You can refer to any component in the codebase to see how it works, e.g. 

https://github.com/ampproject/amphtml/blob/7c8b5ba0e6828b5ed40e038a21d34b46d4437eac/extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js#L181-L220

PS: the image can only be loaded from the `layoutCallback`

Reply:
Done. Load image in `layoutCallback` by copy data-src attribute to src attribute.